### PR TITLE
Pvgna Integration

### PR DESCRIPTION
**Pull request by servercimen**
Added Pvgna links to:
- Player page: Overview tab, Heroes played section
- Player page: Header
- Match page: Overview tab, 
- Match page: Header
- Home page: Sponsored by section

- Icons next to heroes are only displayed if there is relevant content

### CODE REVIEW

File: src/lang/af.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   "app_tracked":"This user has visited recently, and replays of new matches will be automatically parsed.",
   "app_cheese_bought":"Cheese bought",
   "app_dotacoach":"Ask a Coach",
+  "app_pvgna":"Find a Guide",
```
Comment (howardchung) [inline src/lang/af.json]: language strings only need to be added to the english file.  The rest will be translated later.

**Comment by howardchung** [comment]
Looks like a linting issue somewhere

File: src/lang/af.json (hunk 1 above)
Comment (servercimen) [inline src/lang/af.json]: I see, just removed it from everyone except English file.

**Comment by servercimen** [comment]
Just pushed a commit to fix a CSS lint issue. Still, travis shows 2 linting issues in files that isn't changed in this PR.

**Comment by howardchung** [comment]
Yeah, I think it's due to the eslint version bump: https://github.com/odota/ui/pull/666

After that's merged just sync master again and CI should pass.

**Comment by howardchung** [comment]
Looks like a merge conflict with #663 which removed flexbox-grid.

**Comment by servercimen** [comment]
Rebased master, fixed the merge conflict & some UI issues that appeared after rebase. Please let me know if you need anything else to merge this.

File: src/components/Visualizations/Table/HeroImage.jsx [hunk 2]
```diff
@@ -80,6 +84,16 @@ const TableHeroImage = ({
         }
       </div>
     }
+    { !!showPvgnaGuide && pvgnaGuideInfo && heroName &&
```
Comment (howardchung) [inline src/components/Visualizations/Table/HeroImage.jsx]: @servercimen why the double negation?

Comment (servercimen) [inline src/components/Visualizations/Table/HeroImage.jsx]: Sorry for late answer, I think it's not needed here.

I usually use double negation to make sure I have a boolean value (e.g. when the variable can be null.) for logical operators. But here it doesn't matter for the output.

Comment (howardchung) [inline src/components/Visualizations/Table/HeroImage.jsx]: I find Boolean() more readable if I need to make sure something's a certain type

**Pull request merged**